### PR TITLE
fix: ask moderators to order a ticket for the event.

### PR DESCRIPTION
**Pull request by iamdevvalecha**
As discussed in the meeting ask moderators to get a ticket for the event.

### CODE REVIEW

**Comment by iamdevvalecha** [comment]
The changes:- 
Screenshot from 2021-06-01 23-21-32

File: app/components/forms/events/view/videoroom-form.hbs [hunk 1]
```diff
@@ -170,6 +170,9 @@
       </span>
     {{/each}}
   </div>
+   <div class="ui center aligned secondary large segment mt-8" style="margin-bottom:-10px;border:none">
```
Comment (MeghalBisht) [inline app/components/forms/events/view/videoroom-form.hbs:173]: use classes

File: app/styles/libs/_helpers.scss [hunk 2]
```diff
@@ -146,6 +146,10 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
   max-height: 50px;
 }
 
+.mb-10{
```
Comment (maze-runnar) [inline app/styles/libs/_helpers.scss:149]:
```suggestion
.mb-10 {
```

File: app/components/forms/events/view/videoroom-form.hbs [hunk 3]
```diff
@@ -170,6 +170,9 @@
       </span>
     {{/each}}
   </div>
+   <div class="ui center aligned secondary large segment mt-8" style="mb-10 border:none">
```
Comment (maze-runnar) [inline app/components/forms/events/view/videoroom-form.hbs:173]:
```suggestion
   <div class="ui center aligned secondary large segment mt-8 mb-10" style="border:none">
```

**Comment by maze-runnar** [comment]
> The changes:-
> Screenshot from 2021-06-01 23-21-32

is this the correct way of adding helper text? See how it is added to other places.

**Comment by iamdevvalecha** [comment]
The new changes:- 
Screenshot from 2021-06-03 12-21-05

**Review by mariobehling — DISMISSED** [review]
Looks good to me. Thanks

**Review by maze-runnar — DISMISSED** [review]

**Review by MeghalBisht — APPROVED** [review]

**Pull request merged**